webapp/src/ts/modals/edit-message-group/edit-message-group.component.ts

```diff
@@ -78,42 +69,42 @@ export class EditMessageGroupComponent extends MmModalAbstract
             },
           },
           (date) => {
-            const taskId = $(element).data('task-id');
-            const task = this.model.group.rows.find(task => task.task_id === taskId);
             task.due = date.toISOString();
           }
         );
-        this.datePickers[taskId] = $(element).data('daterangepicker');
+        this.datePickers[index] = $element.data('daterangepicker');
       });
     });
   }
 
-  private removeDatePickers(taskId?) {
-    if (taskId) {
-      this.datePickers[taskId]?.remove();
-      delete this.datePickers[taskId];
+  private removeDatePickers(index?) {
+    if (typeof index !== 'undefined') {
```

Comment:
```suggestion
    if (index !== undefined) {
```

In ECMAScript 5 it is safe to compare a variable with `undefined`, not sure when is transpiled if we target a lower standard, but I think no.